### PR TITLE
chore: update extism-runtime.wasm

### LINKED ISSUE #69: Specify the runtime version clearly
When updating the runtime by [simply replacing a Wasm binary file](https://github.com/extism/go-sdk/commits/main/extism-runtime.wasm), I think it would be helpful to clearly include the version number in the update. This way, the version number can be quickly identified without having to run code or dig through commit messages.

### PR DESCRIPTION
**Pull request by hellodword**
add RuntimeVersion

closes #69

### CODE REVIEW

File: extism.go [hunk 1]
```diff
@@ -23,6 +23,9 @@ import (
 //go:embed extism-runtime.wasm
 var extismRuntimeWasm []byte
 
+//go:embed extism-runtime.wasm.version
+var ExtismRuntimeWasmVersion string
```
Comment (nilslice) [inline extism.go:27]: is it possible to make this a const instead? and if not, can we make the name `extismRuntimeWasmVersion` so it is not mutable by any thing that imports it?

If you need a public getter, also feel free to add a function that reads and returns the value.

**Comment by bhelx** [comment]
Thanks for taking the initiative on it!

**Review by zshipko — APPROVED** [review]

**Comment by nilslice** [comment]
Do we have anything else teed up to get into a release? If not, I can cut one so this is available in the go docs & via easier `go get` install

**Comment by zshipko** [comment]
Nothing immediate, I'm hoping to finish this: https://github.com/extism/go-sdk/pull/70 by the end of the week but I can just make another release

**Pull request merged**